### ISSUE TITLE
change image icon

### ISSUE THREAD

**Issue by ccarruitero** [mozillahispano/mozbuzz-share#3]
?

**Comment by nukeador** [comment]
Suggestions?

@bernimel we don't have a logo for mozbuzz, but maybe we could use the same icon as the mozilla hispano favicon (for now).

**Comment by bernimel** [comment]
You're right @nukeador. I don't have the files to work with but maybe we could build something with that M and the Buzz word. See attached image as reference.

![Imagen 2](https://f.cloud.github.com/assets/1993933/14703/e9d3ed80-4672-11e2-8f96-baf5e1163a4d.png)

**Comment by ccarruitero** [comment]
I think is ok if we use mozhispano gecko for now. Exists favicon about this?

**Comment by nukeador** [comment]
@ccarruitero if you reduce the gecko to fit in 16x16, it will be impossible to what it is.

+1 to creating an image "MBuzz" (using open sans font of course)

**Comment by bernimel** [comment]
@ccarruitero where are you going to use the image you're asking for in this issue?

**Comment by nukeador** [comment]
@bernimel We want to use it as a little icon next to the bookmark star on Firefox url bar.

**Comment by bernimel** [comment]
Thanks. Then I guess the horizontal version of my proposal should fit better. It's a very small icon and we will want people to understand/see something.
